operators/src/processing/expression/mod.rs

```diff
@@ -43,7 +43,7 @@ pub struct ExpressionParams {
     #[serde(with = "float_with_nan")]
     pub output_no_data_value: f64, // TODO: check value is valid for given output type during deserialization
     pub output_measurement: Option<Measurement>,
-    pub map_no_data: bool,
+    pub map_no_data: bool, // TODO: remove since that has no use anymore!
```

Comment:
is that so? I think the param specified, whether the expression is evaluated at all if there is a single no data pixel among all inputs and this hasn't changed.

Reply:
is that so? then we can keep it.